### PR TITLE
API-000 veteran verification removal

### PR DESCRIPTION
**Pull request by dmekala-va**
## Summary

- This PR is to remove veteran verification files and rspecs in preparation for deleting the openId_application controller. These files use the openid_application controller therefore the goal is to remove all files that include reference to it and ultimately remove the file. 
- Lighthouse Team Pivot!

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#[API-20648]
- https://jira.devops.va.gov/browse/API-20648
- This relates to the above links and in preparation for the story

## Testing done

- Deleting rspecs and relevant vcr cassettes not used anymore

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jperk51 — CHANGES_REQUESTED** [review]
I'm not really sure I understand what's going on here, but deleting stuff from the mobile module seems dangerous without adding and rerouting to the replacement logic. Have we confirmed that none of the veterans_verification logic is currently being used in prod?

**Review by kpethtel — COMMENTED** [review]
To @jperk51 's point, can you tell us which part of our user request spec is problematic? Perhaps we can just remove that and not the entire file.

**Comment by dmekala-va** [comment]
> To @jperk51 's point, can you tell us which part of our user request spec is problematic? Perhaps we can just remove that and not the entire file.



> To @jperk51 's point, can you tell us which part of our user request spec is problematic? Perhaps we can just remove that and not the entire file.

Yes np. Will add that file back in and modify to fit the veteran verification file removals

**Comment by dbrown-va** [comment]
The `modules/mobile/spec/request/user_request_spec.rb` modification was reverted.  

This change is isolated to the Veteran Verification V0 module removal which has been completed removed from Lighthouse routing and all traffic confirmed to the DVP deployed microservice.

**Review by crolarlibertyva — DISMISSED** [review]

**Review by crolarlibertyva — APPROVED** [review]

File: app/controllers/v0/rated_disabilities_controller.rb [hunk 1]
```diff
@@ -1,34 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline app/controllers/v0/rated_disabilities_controller.rb:1]: This file is not related to the lighthouse module and should not be removed

File: app/controllers/v0/rated_disabilities_discrepancies_controller.rb [hunk 2]
```diff
@@ -1,72 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline app/controllers/v0/rated_disabilities_discrepancies_controller.rb:1]: This file is not related to the lighthouse module and should not be removed

File: lib/disability_compensation/factories/api_provider_factory.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require 'disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider'
-require 'disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider'
```
Comment (jerekshoe) [inline lib/disability_compensation/factories/api_provider_factory.rb:4]: This file is not related to the lighthouse module and should not be removed

File: lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb [hunk 4]
```diff
@@ -1,66 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb:1]: This file is not related to the lighthouse module and should not be removed

File: lib/lighthouse/veteran_verification/configuration.rb [hunk 5]
```diff
@@ -1,126 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline lib/lighthouse/veteran_verification/configuration.rb:1]: This file is not related to the lighthouse module and should not be removed

File: spec/controllers/v0/disability_compensation_forms_controller_spec.rb [hunk 6]
```diff
@@ -1,94 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline spec/controllers/v0/disability_compensation_forms_controller_spec.rb:1]: This file is not related to the lighthouse module and should not be removed

File: spec/lib/lighthouse/veteran_verification/service_spec.rb [hunk 7]
```diff
@@ -1,54 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline spec/lib/lighthouse/veteran_verification/service_spec.rb:1]: This file is not related to the lighthouse module and should not be removed

File: spec/models/form526_submission_spec.rb [hunk 8]
```diff
@@ -1,963 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline spec/models/form526_submission_spec.rb:1]: This file is not related to the lighthouse module and should not be removed

File: spec/requests/v0/disability_compensation_in_progress_forms_controller_request_spec.rb [hunk 9]
```diff
@@ -1,248 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline spec/requests/v0/disability_compensation_in_progress_forms_controller_request_spec.rb:1]: This file is not related to the lighthouse module and should not be removed

File: spec/requests/disability_compensation_form_request_spec.rb [hunk 10]
```diff
@@ -1,373 +0,0 @@
-# frozen_string_literal: true
```
Comment (jerekshoe) [inline spec/requests/disability_compensation_form_request_spec.rb:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/200_Not_Connected_response.yml [hunk 11]
```diff
@@ -1,107 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/200_Not_Connected_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/200_response.yml [hunk 12]
```diff
@@ -1,108 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/200_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/400_response.yml [hunk 13]
```diff
@@ -1,65 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/400_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/401_response.yml [hunk 14]
```diff
@@ -1,64 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/401_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/403_response.yml [hunk 15]
```diff
@@ -1,64 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/403_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/404_ICN_response.yml [hunk 16]
```diff
@@ -1,94 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/404_ICN_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/404_response.yml [hunk 17]
```diff
@@ -1,58 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/404_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/405_response.yml [hunk 18]
```diff
@@ -1,64 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/405_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/413_response.yml [hunk 19]
```diff
@@ -1,62 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/413_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/429_response.yml [hunk 20]
```diff
@@ -1,62 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/429_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/500_response.yml [hunk 21]
```diff
@@ -1,65 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/500_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/502_response.yml [hunk 22]
```diff
@@ -1,94 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/502_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/503_response.yml [hunk 23]
```diff
@@ -1,69 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/503_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/504_response.yml [hunk 24]
```diff
@@ -1,66 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/disability_rating/504_response.yml:1]: This file is not related to the lighthouse module and should not be removed

File: spec/support/vcr_cassettes/lighthouse/veteran_verification/show/200_response.yml [hunk 25]
```diff
@@ -1,129 +0,0 @@
----
```
Comment (jerekshoe) [inline spec/support/vcr_cassettes/lighthouse/veteran_verification/show/200_response.yml:1]: This file is not related to the lighthouse module and should not be removed

**Review by jerekshoe — CHANGES_REQUESTED** [review]

**Comment by dmekala-va** [comment]
Added back all files not related to VV module removal

**Review by crolarlibertyva — APPROVED** [review]

**Comment by dmekala-va** [comment]
Would I be able to get another review please? @jerekshoe and @jperk51

**Review by rjohnson2011 — APPROVED** [review]

**Review by crolarlibertyva — APPROVED** [review]

**Review by jperk51 — APPROVED** [review]

**Review by kpethtel — APPROVED** [review]

**Review by jerekshoe — DISMISSED** [review]

**Pull request closed without merging**